_includes/neurips/2019/about.neurips.2019.html

```diff
@@ -50,38 +50,47 @@ <h2>Workshop Submission Management</h2>
 
       </div>
       <!-- <hr> -->
-      <!-- <div id="nipsProgram">
+      <div id="nipsProgram">
           <h2>Program</h2>
           <h3>Highlights</h3>
           <ul>
-            <li>7:30 - 8:00 am | Check-In </li>
-            <li>8:00 - 8:15 am | Opening Remarks</li>
-            <li>8:15 - 9:00 am | Keynote</li>
-            <li>9:00 - 11:00 am | Oral Presentations</li>
-            <li>11:00 - 12:00 pm | Research Proposal Mentoring Hour & Poster Sessions</li>
-            <li>12:00 - 1:00 pm | Lunch Break</li>
-            <li>1:00 - 2:00 pm | Poster Sessions</li>
-            <li>2:00 - 6:00 pm | Oral Presentations</li>
-            <li>6:30 -7:30 pm | Networking & Roundtable Discussion</li>
+            <li>8:00 - 8:45 am | Registration and Check-In </li>
+            <li>8:45 - 9:00 am | Opening Remarks</li>
+            <li>9:00 - 9:30 am | Keynote</li>
+            <li>9:30 - 10:30 am | Oral Presentations</li>
+            <li>10:30 - 11:15 am | Coffee Break</li>	
+	    <li>11:15 - 11:45 am | Keynote</li>
+            <li>11:45 - 12:45 pm | Oral Presentations</li>
+            <li>12:45 - 1:15 pm | Lunch Google Keynote</li>
+            <li>1:15 - 2:45 pm | Lunch</li>
+            <li>2:45 - 3:15 pm | Keynote</li>
+            <li>3:15 - 3:45 pm | Oral Presentations</li>
+            <li>3:45 - 4:00 pm | Break</li>
+	    <li>4:00 - 5:00 pm | Research Mentoring Hour</li>	
+	    <li>5:00 - 6:00 pm | Roundtable Discussion/ Panel</li>	
+	    <li>6:00 - 8:00 pm | Reception and Poster session</li>	
           </ul>
-          <a class="project-link" href="#">Detailed Schedule</a>
-      </div> -->
+          <!-- <a class="project-link" href="#">Detailed Schedule</a> -->
+      </div>
       <hr>
         <div id="nipsSponsors">
           <h2>Sponsors</h2>
           <p>If you are interested in becoming a sponsor, please check out our
               <a href="/neurips-2019-sponsor">Sponsor Prospectus</a>
-g              and we will follow up with more information.</p>
+              and we will follow up with more information.</p>
           <a class="project-link" href="/neurips-2019-sponsor">Sponsor Prospectus</a>
           <div class='nipsSponsorGallery'>
               <div class='sponsorList' id='platLevel'>
                 <h3>Platinum</h3>
-                  <!-- <a target="_blank" href="https://www.facebook.com"><img width=500
-                    src="./assets/img/icml/2019/sponsors/Facebook.png"></a> -->
+                  <a target="_blank" href="https://www.facebook.com"><img width=500
+                    src="./assets/img/icml/2019/sponsors/Facebook.png"></a>
                   </br>
 		  <a target="_blank" href="https://www.google.com"><img width=500
                     src="./assets/img/icml/2019/sponsors/Gold-Google.png"></a>
                   </br>
+		  <a target="_blank" href="https://www.apple.com"><img width=500
```

Comment:
Apple should be under a silver sponsor heading and the image may need to be resized, currently, it appears stretched when you view this change locally. It's preferable to make a copy of the image from the ICML folder and place it in the NeurIPS sponsor's folder for editing that's required for this specific workshop.

Reply:
I made it half size and created the silver header